src/testsuite/arith128_print.c

```diff
@@ -630,6 +630,29 @@ print_vint256 (char *prefix, vui128_t val0_128, vui128_t val1_128)
 #endif
 }
 
+void
+print_vb128c (char *prefix, vb128_t val)
+{
+  const vui64_t true = { 'T', 'T' };
+  const vui64_t false = { 'F', 'F' };
+  vui64_t text;
+
+  text = vec_sel (false, true, (vb64_t) val);
+
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+  printf ("%s %c%c\n", prefix, (int) text[1], (int) text[0]);
+#else
+  printf ("%s %c%c\n", prefix, (int)text[0], (int)text[1]);
```

Comment:
nit: whitespace differences between this line and 2 lines up (space after cast, or not).